<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listItemWidget</class>
 <widget class="QWidget" name="listItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>42</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>42</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>42</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>listItemWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:rgba(0,0,0,0.7);
}
#btn_delete{
border-image:url(:/masterCashier/Resources/delete.png);
}
#btn_delete:pressed{
margin:1px;
}
#btn_reduce{
border-image:url(:/masterCashier/Resources/reduce_normal.png);
}
#btn_reduce:hover{
border-image:url(:/masterCashier/Resources/reduce_hover.png);
}
#btn_reduce:pressed{
border-image:url(:/masterCashier/Resources/reduce_pressed.png);
}
#btn_add{
border-image:url(:/masterCashier/Resources/add_normal.png);
}
#btn_add:hover{
border-image:url(:/masterCashier/Resources/add_hover.png);
}
#btn_add:pressed{
border-image:url(:/masterCashier/Resources/add_pressed.png);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="btn_delete">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lbl_goodsName">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_stockNum">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_reduce">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="lbl_num">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lbl_sum">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
